pegasus/config.ru

```diff
@@ -1,5 +1,9 @@
 require File.expand_path('../router', __FILE__)
 
+require 'rack/csrf'
+use Rack::Session::Cookie, secret: (CDO.sinatra_session_secret || 'dev_mode')
```

Comment:
This is a new property I have defined in the production envivronment.  You can see it using "knife environment edit production".